test/testfloat/CMakeLists.txt

```diff
@@ -26,6 +26,50 @@ if(TESTFLOAT_GEN)
     if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
         # TODO: Clang produces -0 for input 0, see https://bugs.llvm.org/show_bug.cgi?id=47393.
         list(APPEND IGNORE_LIST ui64_to_f64/min)
+    elseif(CMAKE_CROSSCOMPILING AND CMAKE_SYSTEM_PROCESSOR STREQUAL arm64)
+        # These tests fails on arm64 emulation. TODO: Check if NaNs are the reason.
```

Comment:
@chfast you mentioned these tests are more strict than wasm requires them, but okay on x86. Perhaps mention that here so it is not "misleading" that arm is just failing expected things.

Reply:
I disabled NaN checking for these.